### ISSUE TITLE
network error-Cody unreachable_I always need to login manually eachtime I start VSCode

### ISSUE THREAD

**Issue by dasyari** [sourcegraph/cody#6459]
Type: <b>Bug</b>

## Extension Information

- Cody Version: 1.53.1734972437
- VS Code Version: 1.95.3
- Extension Host: desktop

##  Steps to Reproduce

1.
2.
3.

## Logs

Extension version: 1.53.1734972437
VS Code version: Code 1.95.3 (f1a4fb101478ce6ec82fe9627c43efbf9e98c813, 2024-11-13T14:50:04.152Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz (12 x 2208)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|7.85GB (0.40GB free)|
|Process Argv|--crash-reporter-id c3c4f935-db1e-448d-8b5f-f6c492bba053|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593:30376534
py29gd2263:31024239
vscaac:30438847
c4g48928:30535728
azure-dev_surveyone:30548225
a9j8j154:30646983
962ge761:30959799
pythonnoceb:30805159
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
dvdeprecation:31068756
dwnewjupyter:31046869
newcmakeconfigv2:31071590
nativerepl1:31139838
pythonrstrctxt:31112756
nativeloc2:31192216
cf971741:31144450
iacca1:31171482
notype1:31157159
5fd0e150:31155592
dwcopilot:31170013
stablechunks:31184530
6074i472:31201624

```

</details>

**Comment by dasyari** [comment]
Every time I start VSCode, I always have to login manually, because cody always disconnected: Network error-Cody is unreachable. Thus, I have to login manually each time. Setting it from command:Cody switch account, input URL and token also not working. URL I had tried: 
https://sourcegraph.com/
https://sourcegraph.com/sign-in?returnTo=/cody/manage -->it respond error in firefox browser: "Unexpected Application Error!
error loading dynamically imported module: https://sourcegraph.com/.assets/chunks/chunk-FuzzyFinder-XSCPASO5.js" and refer me to address: "https://sourcegraph.com/cody/manage"
https://sourcegraph.com/cody/manage
These 3 links above is not working.

**Comment by PriNova** [comment]
Hey @dasyari 

Are you using any VPN or proxy services? 
Can you please check if you have too much API access tokens generated in your Sourcegraph client area? (Delete all not needed)

Another issue could be keychaining issue. You can read more about here and how to solve it: https://sourcegraph.com/docs/cody/troubleshooting#signin-fails-on-each-vs-code-restart.

Sorry for the inconvenience,

**Comment by dasyari** [comment]
@PriNova No, I don't use VPN or proxy services. I will try to solve it using that link you gave me. I hope it works.

**Comment by Kynlos** [comment]
Please re-open this thread if the issue still persists in the latest build(s) of Cody.

Thanks,

Kynlo

**Comment by PriNova** [comment]
@dasyari We had a new insight, which might maybe relate to your issue. Do you have any antivirus programs installed, which might block the login process? 
Kapersky does this with a feature to prevent URL injection. Please try to deactivate your virus protection during login. If that helps, then it will be flagged as false positive.

**Comment by PriNova** [comment]
Hey @TanmayBhayani 

Thank you for reporting. The team will have a look at it.

**Comment by PriNova** [comment]
We currently have an unexpected service downtime. For updates of the incident, please have a look at the [Sourcegraph status page](https://sourcegraphstatus.com/).

Sorry for the inconvenience.
